packages/webview_flutter/webview_flutter_android/lib/src/android_webview_controller.dart

```diff
@@ -1268,6 +1268,29 @@ class AndroidNavigationDelegate extends PlatformNavigationDelegate {
           callback(AndroidUrlChange(url: url, isReload: isReload));
         }
       },
+      onReceivedHttpAuthRequest: (
+        android_webview.WebView webView,
+        android_webview.HttpAuthHandler httpAuthHandler,
+        String host,
+        String realm,
+      ) {
+        final void Function(HttpAuthRequest)? callback =
+            weakThis.target?._onHttpAuthRequest;
+        if (callback != null) {
```

Comment:
Don't we need a default reply when there's no handler like we have on iOS?

Reply:
That's a good catch. I updated the PR